domains/transfer-agreements/TransferAgreements.API/API/Data/TransferAgreement.cs

```diff
@@ -1,22 +1,16 @@
 using System;
+using System.Collections.Generic;
 
 namespace API.Data;
 
 public class TransferAgreement
 {
     public Guid Id { get; set; }
-
     public DateTimeOffset StartDate { get; set; }
-
     public DateTimeOffset? EndDate { get; set; }
-
-    public string ActorId { get; set; }
-
     public Guid SenderId { get; set; }
-
     public string SenderName { get; set; }
-
     public string SenderTin { get; set; }
-
     public string ReceiverTin { get; set; }
+    public ICollection<TransferAgreementHistoryEntry>? HistoryEntries { get; set; }
```

Comment:
Is it possible to remove this collection? And also remove the property `TransferAgreement` on TransferAgreementHistoryEntry?

Reply:
Entity framework uses that field to generate the foreign key in our HistoryEntries table, so i cannot remove the TransferAgreementProperty. 

I have removed the Icollection, though. as that is not necessary.